system/Model.php

```diff
@@ -118,6 +118,13 @@ class Model extends BaseModel
      */
     protected $escape = [];
 
+    /**
+     * Primary Key data when inserting and useAutoIncrement is false.
+     *
+     * @var int|string|null
+     */
+    private $tempPrimaryKeyData;
```

Comment:
Why "data"? If differentiating from the name I think "value" would be more consistent with what we usually do for a single item.

Reply:
Yes, *value* is better.
Fixed.